application/Modules/Invoices/Models/ItemAmount.php

```
@@ -27,10 +27,14 @@ class ItemAmounts extends CI_Model
      * @param $item_id
      * @param $global_discount
+     *
+     * Legacy migration info:
+     * @legacy-file application/modules/invoices/models/Mdl_item_amount.php
+     * @legacy-function calculate()
      */
     public function calculate($item_id, &$global_discount)
     {
-        $this->load->model('invoices/mdl_items');
+        $this->load->model('invoices/item');
         $item = $this->mdl_items->get_by_id($item_id);
```

PHPMD (2.15.0)

The variable $item_id is not named in camelCase. (undefined)

(CamelCaseVariableName)